apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -56,6 +55,19 @@ type FilesComponentProps = {
   setFileType: setFileType;
 };
 
+const extractPrompt = (
+  promiseResults: DialogClosePromiseReturnType
+): string => {
+  const {type, args} = promiseResults;
+  if (type === 'confirm') {
+    return args as string;
+  }
+
+  return '';
+};
```

Comment:
Given a promise from the dialog close, we know that the `args` will be a string in the prompt. Return that.

If the user didn't click confirm, return an empty string.

Reply:
mind putting this in a comment? Also see my comment below about being confused about what `prompt` means.